lms/templates/courseware/progress_graph.js

```diff
@@ -143,9 +219,10 @@ $(function () {
     
     %if show_grade_breakdown:
       var o = plot.pointOffset({x: ${overviewBarX} , y: ${totalScore}});
-      $grade_detail_graph.append('<div style="position:absolute;left:' + (o.left - 12) + 'px;top:' + (o.top - 20) + 'px">${"{totalscore:.0%}".format(totalscore=totalScore)}</div>');
+      $grade_detail_graph.append('<div style="position:absolute;left:' + (o.left - 12) + 'px;top:' + (o.top - 20) + 'px"><span class="sr">Overall grade:</span> ${"{totalscore:.0%}".format(totalscore=totalScore)}</div>');
```

Comment:
Yeah :\ I created a ticket to i18n this whole file. It didn't have any before, and I haven't added any in this round.